jdaviz/configs/specviz/helper.py

```diff
@@ -41,11 +41,35 @@ def __init__(self, *args, **kwargs):
         self.app.hub.subscribe(self, RedshiftMessage,
                                handler=self._redshift_listener)
 
+    @deprecated(since="3.6", alternative="specviz.load_data")
     def load_spectrum(self, data, data_label=None, format=None, show_in_viewer=True,
                       concat_by_file=False):
         """
         Loads a data file or `~specutils.Spectrum1D` object into Specviz.
 
+        Parameters
```

Comment:
Isn't it a little too late the improve `load_spectrum` doc now? 😆  Doesn't hurt though...